payments-core/src/main/java/com/stripe/android/view/CardWidgetViewModel.kt

```diff
@@ -85,7 +82,7 @@ internal class CardWidgetViewModel(
             return CardWidgetViewModel(
                 paymentConfigProvider = { PaymentConfiguration.getInstance(context) },
                 stripeRepository = stripeRepository,
-                handle = extras.createSavedStateHandle()
+                handle = SavedStateHandle()
```

Comment:
This feels problematic, why are we no longer building the `SavedStateHandle` from the `CreationExtras`? How will we receive the proper `SavedStateHandle` when returning from process death if we create an empty instance every time?

Reply:
I removed `extras.createSavedStateHandle()` because that will cause an exception when we're mocking the view lifecycle owner, registry etc. I tested with "Don't Keep Activities" enabled and it still restores state properly